app/scripts/build-protos.js

```diff
@@ -1,11 +1,22 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const util = require('util');
+const https = require('https');
 const exec = util.promisify(require('child_process').exec);
 const { promises: fs } = require('fs');
 const { join, sep } = require('path');
 const { platform } = require('os');
 const appPath = join(__dirname, '..');
 
+/** Specify the versions of LND and Loop protos to download */
+const LND_VERSION = 'v0.11.1-beta';
+const LOOP_VERSION = 'v0.11.2-beta';
```

Comment:
Great idea checking that we have the correct version of the `*.proto` themselves checked in.
We _could_ even go a step further and parse this version out of the `go.mod` in the root directory. That way we'd notice if we update the dependency that's built into the unified binary without updating the protos.
Can be done it a separate PR though.

Reply:
Great point! I didn't think to use the `go.mod` for the versions but that makes total sense.